python/simulation/fake_raft.py

```diff
@@ -0,0 +1,509 @@
+"""
+Makes a raft's worth of image files based on a user-specified single-sensor
+image file, the raft designator, and the sensor type (ITL or E2V, which 
+probably could be guessable from the single-sensor image).  The test type
+probably also needs to be specified (for the file naming)
+The data in the nine output images are the same as in the input image.  What
+is new is the raft coordinate-related header keywords.  These are derived
+from the specification in LCA-13501.
+"""
+
+from __future__ import print_function, absolute_import, division
+#import os
+#import copy
+import numpy as np
+import astropy.io.fits as fits
+import os
+import yaml
+
+import fnmatch
+import lcatr.schema
+from DataCatalog import DataCatalog
+
+import siteUtils
+
+
+def sensor_type_from_id(sensor_id):
+    """  Get the sensor type ( e2v-CCD or ITL-CCD ) from the sensor_id string
+    """
+    stype =  sensor_id[0:7]
```

Comment:
What LCATR_UNIT_TYPE set to when running tests at the raft level?  Would it actually be a e2v-CCD or ITL-CCD?   Anyway, I changed the code to either get this from the YAML file or from the first sensor in the raft (when using the eTraveler)